MMM-AlarmClock.js

```diff
@@ -123,7 +124,6 @@ Module.register('MMM-AlarmClock', {
     getDom() {
         const wrapper = document.createElement('div');
         const header = document.createElement('header');
-        header.classList.add('align-left');
```

Comment:
For backward compatibility this should be done in custom.css or as a config option. Also this is not related to the original reason for this pr. If you think this is a great addition to the module please create a seperate pr as described. For now i will undo this change.